README.md

```diff
@@ -178,405 +128,133 @@ After installing the library you will need to require it in your test file (see
+### `waitForTextToBePresentInElement`
 
-```js
-const protractorHelper = require("protractor-helper");
+That method waits for a specific text to be present in a specific HTML element.
+[Example](docs/EXAMPLES.md#waitfortexttobepresentinelement)
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+### `waitForTextNotToBePresentInElement`
 
-        protractorHelper.waitForUrlNotToContainString("foobarbaz", "URL contains the string 'foobarbaz'", 3000);
+That method is the opposite of the previous one, so, it waits for a specific text not to be present in a specific HTML element.
+[Example](docs/EXAMPLES.md#waitfortextnottobepresentinelement)
 
-        // ...
-    });
-});
-```
-
-### Example of usage of `fillFieldWithTextWhenVisibleAndPressEnter`
-
-```js
-const protractorHelper = require("protractor-helper");
+### `waitForUrlToBeEqualToExpectedUrl`
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+That method waits for the URL to be equal to an expected URL. Such method is useful when you want to continue performing actions on elements only when in the correct URL.
+[Example](docs/EXAMPLES.md#waitforurltobeequaltoexpectedurl)
 
-        const textField = element(by.css("input.some-text-field"));
+### `waitForUrlNotToBeEqualToExpectedUrl`
 
-        protractorHelper.fillFieldWithTextWhenVisibleAndPressEnter(textField, "some text", "textField is not visible", 3000);
+That method waits for the URL not to be equal to an expected URL. Such method is useful when you want to continue performing actions on elements only when not in a specific URL.
+[Example](docs/EXAMPLES.md#waitforurlnottobeequaltoexpectedurl)
 
-        // ...
-    });
-});
-```
+### `waitForUrlToContainString`
 
-### Example of usage of `scrollToElementWhenVisible`
+That method waits for the URL to contain an expected string. Such method is useful when you want to perform verifications based on the current URL.
+[Example](docs/EXAMPLES.md#waitforurltocontainstring)
 
-```js
-const protractorHelper = require("protractor-helper");
+### `waitForUrlNotToContainString`
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+That method waits for the URL not to contain an expected string. Such method is useful when you want to perform verifications based on the current URL.
+[Example](docs/EXAMPLES.md#waitforurlnottocontainstring)
 
-        const myLink = element(by.css("a.my-link"));
+### `fillFieldWithTextWhenVisibleAndPressEnter`
 
-        protractorHelper.scrollToElementWhenVisible(myLink, "my link is not visible", 3000);
+That method fills an input field with a text as soon as such field is visible and then it simulates pressing the ENTER key from the keyboard. This method is useful in cases such as when doing a search and pressing the ENTER key, instead of having to fill the input field and clicking the search button, for example.
```

Comment:
Same.

Reply:
Commit made because this review: ce8ef21